core/test/utils/api.js

```diff
@@ -149,14 +149,15 @@ const login = (request, API_URL) => {
                 password: 'Sl1m3rson99',
                 client_id: 'ghost-admin',
                 client_secret: 'not_available'
-            }).then(function then(res) {
-            if (res.statusCode !== 200) {
-                return reject(new common.errors.GhostError({
-                    message: res.body.errors[0].message
-                }));
-            }
-
-            resolve(res.body.access_token);
+            })
+            .then(function then(res) {
+                if (res.statusCode !== 200 && res.statusCode !== 201) {
+                    return reject(new common.errors.GhostError({
+                        message: res.body.errors[0].message
+                    }));
+                }
+
+                resolve(res.headers['set-cookie'] || res.body.access_token);
```

Comment:
The target API local utility should pick the result from the response.

Reply:
I haven't done this in my PR FYI

Reply:
No problem :)